mobify/progressive-web-scaffold
# Add Themes.scss

Compile and add `themes.scss` to the scaffold.

## Changes
- Add example theme file
- Add a central stylesheet for themes to be pulled in
- Tweak Webpack configuration to extract two separate CSS files: one for the main stylesheet, one for the themes.

## How to test-drive this PR
- Add some styles to `app/components/example/theme.scss`, like `body { background: red; }`
- Run `npm run dev`
- [Preview](https://preview.mobify.com/?url=http%3A%2F%2Fwww.merlinspotions.com&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Confirm that the theme styles you added apply
